packages/bigcommerce-payments-integration/src/index.ts

```diff
@@ -70,11 +70,11 @@ export { WithBigCommercePaymentsAlternativeMethodsPaymentInitializeOptions } fro
 
 /**
  *
- * BigCommercePayments Klarna methods strategies
+ * BigCommercePayments Redirect Alternative methods strategies
  *
  */
-export { default as createBigCommercePaymentsKlarnaPaymentStrategy } from './bigcommerce-payments-klarna/create-bigcommerce-payments-klarna-payment-strategy';
-export { WithBigCommercePaymentsKlarnaPaymentInitializeOptions } from './bigcommerce-payments-klarna/bigcomemrce-payments-klarna-payment-initialize-options';
+export { default as createBigCommercePaymentsRedirectAlternativeMethodsPaymentStrategy } from './bigcommerce-payments-redirect-alternative-methods/create-bigcommerce-payments-redirect-alternative-methods-payment-strategy';
```

Comment:
Don't forget to update the `integrations` array in `checkout-js` using the renamed method. Take a look here: `BigCommercePaymentsPaymentMethodComponent.tsx`